ydb/core/tx/schemeshard/schemeshard__operation_alter_login.cpp

```diff
@@ -111,13 +114,39 @@ class TAlterLogin: public TSubOperationBase {
                     } else {
                         db.Table<Schema::LoginSidMembers>().Key(removeGroupMembership.GetGroup(), removeGroupMembership.GetMember()).Delete();
                         result->SetStatus(NKikimrScheme::StatusSuccess);
+                        if (response.Warning) {
```

Comment:
AddGroup has `if Notice`, RemoveGroup has `if Warning`. Maybe there should be the same condition?

Reply:
I just made it like in pg, but yeah, maybe it is better to make it with the same levels